packages/icons-scripts/scripts/output/react.js

```diff
@@ -46,6 +48,7 @@ export const ${componentName}: React.FC<${componentName}Props> & ${typeAssigns}
       viewBox={viewBox}
       width={width}
       height={height}
+      style={{color: fill, ...style}}
```

Comment:
```suggestion
      style={fill ? { color: fill, ...style } : style}
```

по мотивам https://github.com/VKCOM/VKUI/pull/7979